extensions/wikia/ExactTargetUpdates/tasks/ExactTargetUserTaskHelper.php

```diff
@@ -156,6 +156,63 @@ public function prepareSubscriberDeleteData( $sUserEmail ) {
 	}
 
 	/**
+	 * Prepares array of params for ExactTarget API for retrieving UserID_WikiID DataExtension objects
+	 * @param array $aUsersIds Array consists of user ids in key. Value doesn't matter.
+	 * e.g. $aUsersIds = [ 321, 12345 ]
+	 * @return array
+	 */
+	public function prepareUserEditsRetrieveParams( array $aUsersIds ) {
+		/* Get Customer Keys specific for production or development */
+		$aCustomerKeys = $this->getCustomerKeys();
+		$sCustomerKey = $aCustomerKeys[ 'UserID_WikiID' ];
+		$aApiParams = [ 'DataExtension' => [] ];
+
+		$aApiParams[ 'DataExtension' ] = [
+			'ObjectType' => "DataExtensionObject[$sCustomerKey]",
+			'Properties' => [ 'user_id', 'wiki_id', 'contributions' ],
+		];
+
+		$sSimpleOperator = 'equals';
+		if ( sizeof( $aUsersIds ) > 1 ) {
```

Comment:
I'd avoid using `sizeof`, in favor of `count`. It's just an alias of `count` (and we use `count` at least close to uniformly), but `sizeof` means something else in C and C++ (it returns the size of a datatype), so it's best to avoid any confusion for people reading your code and stick to the standard `count`.